app/models/user.rb

```diff
@@ -5,7 +5,7 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :trackable, :validatable
   # Hooks
   before_validation :generate_verification_code, on: :create
-  validates :first_name, :last_name, :email, presence: true
+  validates :first_name, :last_name, :email, :locale, presence: true
```

Comment:
Not sure if  `:locale` should be always present :thinking: IMO this isn't a required field, but you can leave it as it is if you want.